examples/complete/main.tf

```
@@ -1,4 +1,4 @@
-module "yaml_config" {
+module "monitor_configs" {
   source = "git::https://github.com/cloudposse/terraform-yaml-config.git?ref=tags/0.1.0"
```

Can we migrate these source references to this new format?

```suggestion
  source  = "cloudposse/yaml/config"
  version = "0.1.0"
```